drivers/platform/nintendo3ds/ctr_spi.c

```diff
@@ -166,10 +169,10 @@ static int ctr_spi_transfer_one(struct spi_master *master,
 
 	if (xfer->tx_buf) {
 		read = false;
-		buf = (u32*)xfer->tx_buf;
+		buf = (u32 *)xfer->tx_buf;
 	} else if (xfer->rx_buf) {
 		read = true;
-		buf = (u32*)xfer->rx_buf;
+		buf = (u32 *)xfer->rx_buf;
```

Comment:
curious spacing in the casts. I wonder if the kernel's clang format config needs to be fixed? cc @ojeda

Reply:
Seems OK to me -- kernel style is `int *x;` instead of `int* x;` or `int * x;`.